.editorconfig

```diff
@@ -0,0 +1,18 @@
+# https://editorconfig.org
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+indent_style = space
+
+[Makefile]
+indent_style = tab
+indent_size = 4
```

Comment:
I would prefer to keep tab size opinions up to the editor.

Reply:
No problem - will remove from config 👍